website/docs/Installation/linux-install.md

```diff
@@ -1,64 +1,131 @@
-```
-After installation, to open Podman Desktop, use the following command:
+* [Flatpak](https://flatpak.org/setup/)
+* [Podman](https://podman.io/whatis.html) stable version
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+#### Procedure
 
-### 2. Using [Flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
+1. Assert the Flathub repository is enabled, and add it if required:
 
-:::infoPrerequisite 
-Verify Flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up Flatpak.
-:::
+    ```shell
+    $ flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
+    ```
 
-Open the Terminal and go to Downloads directory. 
+2. Install Podman Desktop from Flathub:
 
-```sh
-cd Downloads
-```
+    ```shell
+    $ flatpak install --user flathub io.podman_desktop.PodmanDesktop
+    ```
 
-> The Flatpak identifier for the application is `io.podman_desktop.PodmanDesktop`. This is required to install & start the application using Flatpak.
+#### Verification
 
-Run the following command to install the Flatpak application,
+* Open Podman Desktop from a terminal:
 
-```sh
-flatpak install io.podman_desktop.PodmanDesktop
-```
+  ```shell
+  $ flatpak run io.podman_desktop.PodmanDesktop
+  ```
 
-Run the following command to start the application,
+#### Update
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+* Update Podman Desktop from Flathub:
 
-You can read more about running Flatpak applications [here](https://docs.flatpak.org/en/latest/using-flatpak.html).
+  ```shell
+  $ flatpak update --user io.podman_desktop.PodmanDesktop
+  ```
 
-### 3. Using zip file
+#### Additional resources
 
-Download the zip file and extract the folder.
+* [Podman Desktop Flathub package](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
+* [Using Flatpak](https://docs.flatpak.org/en/latest/using-flatpak.html)
 
-Navigate within the folder and double-click on the `podman-desktop` executable file. This should start the application for you.
+#### Next steps
 
-## Next Steps
+* [Getting started](../getting-started/getting-started)
 
-Learn more on how to get started with Podman Desktop by clicking [here](/docs/getting-started/getting-started).
+## Installing Podman Desktop from a Flatpak bundle {#flatpak-bundle}
+
+Consider installing a Flatpak bundle rather than Flathub when:
+
+* You cannot use Flathub.
+* You want to install an unreleased version.
+
+#### Prerequisites
+
+* [Flatpak](https://flatpak.org/setup/)
+* [Podman](https://podman.io/whatis.html) stable version
+
+#### Procedure
+
+1. Download the Flatpak bundle to a `$HOME/Downloads/podman-desktop-<version>.flatpak` file from:
+
+    * [Downloads page](../../../downloads/linux)
+
+    * [Git repository release assets](https://github.com/containers/podman-desktop/releases)
+
+2. Install the Flatpak bundle:
```

Comment:
I don't think I've ever heard of flatpak be called a `Flatpak bundle` it sounds better a singular word. "Install the Flatpak" instead of "Install the Flatpak bundle".

Even the site doesn't reference it as a bundle other than describing it: https://flatpak.org/

Reply:
> Terminology
> [...]
>    Flatpak bundle: a specific single-file export format which contains a Flatpak app or runtime.



https://docs.flatpak.org/en/latest/introduction.html

Reply:
Yes, but when referring to install, it's not a flatpak bundle, it's a flatpak app, see the official flatpak documentation for clarification: https://docs.flatpak.org/en/latest/using-flatpak.html#install-applications 

So it should be `Install the Flatpak` or `Install the Flatpak app` not bundle, as a bundle is when you initially download it.

Reply:
https://docs.flatpak.org/en/latest/single-file-bundles.html?highlight=bundle#single-file-bundles

Reply:
> The file we download has a .flatpak extension, and it is exactly what is described there:
> 
> https://docs.flatpak.org/en/latest/single-file-bundles.html?highlight=bundle#single-file-bundles

Yup, but when referring to an install, it's an app, not a bundle. 

We should be using the official Flatpak terminology and referring to it as an app, as per the example here:
* https://docs.flatpak.org/en/latest/using-flatpak.html#install-applications

Even in our official Red Hat documentation on Flatpak we do not refer to it as a bundle, but an application:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/assembly_installing-applications-using-flatpak_using-the-desktop-environment-in-rhel-8

Reply:
I still think we should say `Flatpak app` and not include the "bundle" terminology in this documentation, as I believe it's incorrect 🤷 .

All of these sources (including our official Red Hat docs) either don't refer to it as a bundle, nor include references to it. The terminology for Flatpak is referring to it as an app.

* https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/assembly_installing-applications-using-flatpak_using-the-desktop-environment-in-rhel-8
* https://opensource.com/article/21/11/install-flatpak-linux
* https://developer.fedoraproject.org/deployment/flatpak/flatpak-usage.html